cmd/ceremony/main.go

```diff
@@ -368,7 +377,62 @@ func openSigner(cfg PKCS11SigningConfig, issuer *x509.Certificate) (crypto.Signe
 	return signer, newRandReader(session), nil
 }
 
-func signAndWriteCert(tbs, issuer *x509.Certificate, subjectPubKey crypto.PublicKey, signer crypto.Signer, certPath string) error {
+func signAndLintCert(tbs, issuer *x509.Certificate, subjectPubKey crypto.PublicKey, realSigner crypto.Signer, skipLints []string) error {
+	var lintSigner crypto.Signer
+	var err error
+	switch k := realSigner.Public().(type) {
```

Comment:
This duplicates some code in signer/signer.go:NewSigner. Could you factor that out into a small `lints` package that has, for now, a function like `lints.MakeSigner(crypto.Signer) (crypto.Signer, error)`?

Actually, it would probably be good to have another function in that package that, given an input linting key, performs the lints, since that code is also somewhat duplicated between here and signer.go.

Reply:
Yeah, agreed. I've extracted it into a separate package and things seem much more readable now. PTAL.